source/particle/property/melt_particle.cc

```diff
@@ -0,0 +1,102 @@
+/*
+  Copyright (C) 2015 - 2016 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file doc/COPYING.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+
+#include <aspect/particle/property/melt_particle.h>
+#include <aspect/simulator.h>
+#include <aspect/melt.h>
+
+namespace aspect
+{
+  namespace Particle
+  {
+    namespace Property
+    {
+      template <int dim>
+      void
+      Melt<dim>::initialize_one_particle_property(const Point<dim> &position,

```

Comment:
please comment out the `position` variable name here as follows:
```
Melt<dim>::initialize_one_particle_property(const Point<dim> &/*position*/, 
```
Otherwise some compilers complain that it is not used within the function.